StrangeIoC/scripts/strange/extensions/reflector/api/IReflectedClass.cs

```diff
@@ -63,14 +62,24 @@ public interface IReflectedClass
 		KeyValuePair<MethodInfo, Attribute>[] attrMethods { get; set; }
 
 		/// [Obsolete"Strange migration to conform to C# guidelines. Removing camelCased publics"]
-		KeyValuePair<Type, PropertyInfo>[] setters{ get; set;}
-		/// [Obsolete"Strange migration to conform to C# guidelines. Removing camelCased publics"]
-		object[] setterNames{ get; set;}
-		/// [Obsolete"Strange migration to conform to C# guidelines. Removing camelCased publics"]
 		bool preGenerated{ get; set;}
 
 
 		bool hasSetterFor(Type type);
 	}
+
+	public struct ReflectedAttribute
+	{
+		public Type type;
+		public object name;
+		public PropertyInfo propertyInfo;
+
+		public ReflectedAttribute(Type type, PropertyInfo propertyInfo, object name )
+		{
+			this.type = type;
+			this.propertyInfo = propertyInfo;
+			this.name = name;
+		}
+	}
```

Comment:
Seems a little odd to have the ReflectedAttribute struct defined inside the interface file.